### PR TITLE
Fix share space edge case where space is public but not invitable

### PR DESCRIPTION
**Pull request by t3chguy**

### CODE REVIEW

File: src/components/views/spaces/SpacePublicShare.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import {copyPlaintext} from "../../../utils/strings";
 import {sleep} from "../../../utils/promise";
 import {RoomPermalinkCreator} from "../../../utils/permalinks/Permalinks";
 import {showRoomInviteDialog} from "../../../RoomInvite";
+import {MatrixClientPeg} from "../../../MatrixClientPeg";
```
Comment (gsouquet) [inline src/components/views/spaces/SpacePublicShare.tsx:26]: Looks like the consensus seems to be with spaces around curlies.
Patentiently waiting for matrix-org/eslint-plugin-matrix-org#6/

**Review by gsouquet — APPROVED** [review]
Otherwise looks good 👍

File: src/components/views/spaces/SpacePublicShare.tsx (hunk 1 above)
Comment (t3chguy) [reply src/components/views/spaces/SpacePublicShare.tsx:26]: Sure, just was staying consistent with the rest of the file, if it was just a couple of wrongly styled imports I'd have replaced them all but leaving a mix is in-ideal

Comment (t3chguy) [reply src/components/views/spaces/SpacePublicShare.tsx:26]: (though this was probably an automatic import so should improve once the eslint rule lands)

**Pull request merged**